test/bioprinttest.c

```diff
@@ -11,8 +11,12 @@
 
 #include <stdio.h>
 #include <string.h>
-#include <openssl/opensslconf.h>
-#include OPENSSL_UNISTD
+#include <openssl/e_os2.h>
+#ifdef OPENSSL_SYS_WINDOWS
+# include <winsock.h>
```

Comment:
> I'd argue that this is unjustified. There is no reason for test program to "step outside" POSIX. write(2) is not POSIX and shouldn't be used.

I meant standard C library as specified by C language standard, not POSIX. write(2) is part of POSIX, but not standard C library. Sorry about confusion...